starlight/templates/profile_views/edit_competency.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% load i18n %}
+
+{% block content %}
+
+<h1 class="ui black image header">
+    <div class="header">
+        {%  trans "Competency: " %}{{competency.skill}}
+    </div>
+</h1>
+
+<div class="ui form">
+    <form action="." method="post" class="ui large form" id="competency_form">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
I wasn't able to figure out how to load jquery from node_modules instead of from online